app/database/itemStore.go

```diff
@@ -238,7 +243,7 @@ func (s *ItemStore) GetRawItemData(rootID, userID, userLanguageID int64, user Au
 
 // AccessRights returns a composable query for getting
 // (idItem, fullAccess, partialAccess, grayedAccess, accessSolutions) for the given user
-func (s *ItemStore) AccessRights(user AuthUser) *DB {
+func (s *ItemStore) AccessRights(user *User) *DB {
```

Comment:
I supposed `AccessRights(..)` was already there. 🤨... because I find `AccessRight` quite a bad name for what it does. It should return the access right for such a name while... it applies access right? hmm 🤔 Isn't it the previously-called "visible"? 
When reading a use of this function, this is quite confusing.

Reply:
How would you like to rename it?
btw, 'Visible' is there too: ```// Visible returns a view of the visible items for the given user
func (s *ItemStore) Visible(user *User) *DB {
```

Reply:
WithAccessRights?

Reply:
Ok, then I just misunderstood what it was :-)
I'm rechecking what it does exactly.

Reply:
Mmh right, that's not "visible" at all, not "WithAccessRights" as there is no items, ... mmh still `items.AccessRight(user)` look strange to me as AccessRights of items (and still, it is only id+perm) is not very clear. 
Maybe being on `GroupItems` would make more sense (it is not the item itself)... with a function such as `AccessRightByItemID(user)` ?
Ok, anyway, it is not really related with this PR, it can be done separetely, sorry to bother you with that one.